src/samples/ProofOfEntitlement/ProofOfEntitlement.tsx

```diff
@@ -54,26 +69,21 @@ export default function ProofOfEntitlement() {
             // If no claimant data in response, assume no award (or api error)
             if (result.IsAPIError) {
               setShowProblemWithService(true);
+              setPageContentReady(true);
```

Comment:
we can add setPageContentReady(true); in then block

Reply:
Nice Catch, Implemented the fix :)